plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibraries.java

```diff
@@ -173,4 +173,12 @@ private static void resolveTransitiveDependencies(ImmutableMap<String, Library>
       library.setLibraryDependencies(transitiveDependencies);
     }
   }
+
+  /**
+   * @return
+   */
+  public static Object getAllLibraries() {
+    // TODO Auto-generated method stub
```

Comment:
do you want to fill this in? or at least remove the auto boilerplate?

Reply:
Oops.